### ISSUE TITLE
dark theme: text file preview is boxed and inverted

### ISSUE THREAD

**Issue by sndrr** [nextcloud/server#19596]
### Steps to reproduce
1. enable NC dark mode in `settings` > `accessability`
2. go back to file mode
3. find a .txt file, and press the three dots
4. select the (i) details
5. observe that the file preview is boxed in with white background and black text

### Expected behaviour
White text on a black background, unboxed, like it shows in NC18:
![image](https://user-images.githubusercontent.com/5821198/75100069-4acda500-55c9-11ea-83f0-20eb781dcf68.png)

### Actual behaviour
Black text on a white background
![image](https://user-images.githubusercontent.com/5821198/75100021-b95e3300-55c8-11ea-8b9e-f6b806f2422e.png)

### Server configuration

**Operating system:**
Linux

**Web server:**
Nginx

**Database:**
Mysql

**PHP version:**
7.4

**Nextcloud version:** (see Nextcloud admin page)
19.0.0 alpha

**Updated from an older Nextcloud/ownCloud or fresh install:**
fresh

**Where did you install Nextcloud from:**
git

**Signing status:**
<details>
<summary>Signing status</summary>

```
Integrity checker has been disabled. Integrity cannot be verified.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.5.0
  - cloud_federation_api: 1.2.0
  - comments: 1.9.0
  - dav: 1.15.0
  - federatedfilesharing: 1.9.0
  - federation: 1.9.0
  - files: 1.14.0
  - files_sharing: 1.11.0
  - files_trashbin: 1.9.0
  - files_versions: 1.12.0
  - lookup_server_connector: 1.7.0
  - mail: 1.1.2
  - oauth2: 1.7.0
  - provisioning_api: 1.9.0
  - settings: 1.1.0
  - sharebymail: 1.9.0
  - systemtags: 1.9.0
  - theming: 1.10.0
  - twofactor_backupcodes: 1.8.0
  - updatenotification: 1.9.0
  - workflowengine: 2.1.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - contacts
  - encryption
  - files_external
  - testing
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nc.sanderr.dev"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "19.0.0.0",
        "overwrite.cli.url": "https:\/\/nc.sanderr.dev",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "mysql.utf8mb4": true
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

### Client configuration
**Browser:**
Opera, Firefox, Chrome

**Operating system:**
Windows

### Logs
n/a

**Comment by szaimen** [comment]
I could reproduce this on NC21.0.2
cc @nextcloud/javascript for pointers where this needs to be fixed in the code.

**Comment by gary-kim** [comment]
That is an image so we will either need to change the generation or just invert the picture. Inverting the picture feels really janky so if we change the generation, that'd be more of a PHP fix.

**Comment by skjnldsv** [comment]
This is a wontfix imho.
Previews are generated at once. So We cannot generate different dark/light previews.
Documents are on a white page, this is is a preview of the document like a paper sheet.
